src/transformers/modeling_tf_utils.py

```diff
@@ -460,8 +460,8 @@ def generate(
         input_ids=None,
         max_length=None,
         min_length=None,
-        do_sample=True,
-        early_stopping=False,
+        do_sample=None,
```

Comment:
you mean False?

Reply:
The docstrings say it defaults to False.

Reply:
I like it better like this, because all inputs are set no `None` . It defaults to `False` because that's how it's set in `self.config.do_sample` as a default. But up for debate I guess.

Reply:
I'm good with that. Probably better not have several places to setup the "defaults" indeed.
Maybe just indicate in the docstring that the defaults are set in `PretrainedConfig`?